src/app/create/pipelines/pipelines.component.html

```diff
@@ -1,5 +1,14 @@
 <div id="pipelines" class="content container">
-  <alm-toolbar [config]="toolbarConfig" (onFilterChange)="filterChange($event)" (onSortChange)="sortChange($event)"></alm-toolbar>
-  <hr />
+  <alm-toolbar [config]="toolbarConfig"
+               [viewsTemplate]="addOpenShiftConsoleLink"
+               (onFilterChange)="filterChange($event)"
+               (onSortChange)="sortChange($event)">
+    <ng-template #addOpenShiftConsoleLink>
```

Comment:
This looks wrong but I want to move this forward. @dlabrecq can you please submit an issue with directions on this, please.